inference-engine/src/vpu/graph_transformer/src/frontend/detect_network_batch.cpp

```diff
@@ -27,27 +27,20 @@ void FrontEnd::detectNetworkBatch(
     VPU_PROFILE(detectNetworkBatch);
 
     const auto& env = CompileEnv::get();
-    ie::details::CaselessEq<std::string> cmp;
-
-    env.log->trace("Detect network batch");
-    VPU_LOGGER_SECTION(env.log);
-
-    const auto batchSize = network.getBatchSize();
-    env.log->trace("Batch size = %d", batchSize);
 
     auto checkForDeprecatedCnn = [&network, &env]() {
         return !network.getFunction()
                && !env.config.forceDeprecatedCnnConversion
                && !dynamic_cast<const ie::details::CNNNetworkImpl*>(&network);
     };
-    VPU_THROW_UNLESS(!checkForDeprecatedCnn(), "Unexpected CNNNetwork format: it was converted to deprecated format prior plugin's call");
 
-    if (batchSize == 1 || !env.config.detectBatch) {
-        env.log->trace("Keep original network");
+    if (!env.config.detectBatch || checkForDeprecatedCnn()) {
```

Comment:
So now in case of unexpected nGraph -> CNN conversion we're not going to throw an exception. Could you please explain the intention?

I have an impression that even if user sets `forceDeprecatedCnnConversion` it'd fail

Reply:
Because old reshape is removed and code below will definitely fail. So this check is not really important anymore.

Reply:
So should we remove this?